kahpp-spring-autoconfigure/src/main/java/dev/vox/platform/kahpp/actuator/KafkaStreamsState.java

```diff
@@ -0,0 +1,36 @@
+package dev.vox.platform.kahpp.actuator;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.kafka.streams.KafkaStreams;
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.Status;
+import org.springframework.kafka.config.StreamsBuilderFactoryBean;
+import org.springframework.stereotype.Component;
+
+@Component
+public class KafkaStreamsState extends AbstractHealthIndicator {
+
+  private final List<StreamsBuilderFactoryBean> streamsBuilders;
+
+  public KafkaStreamsState(List<StreamsBuilderFactoryBean> streamsBuilders) {
+    this.streamsBuilders = List.copyOf(streamsBuilders);
+  }
+
+  @Override
+  @SuppressWarnings("PMD.CloseResource")
+  protected void doHealthCheck(Health.Builder builder) {
+    Map<String, Status> streamsHealthMap = new HashMap<>();
```

Comment:
With the current implementation, streamsHealthMap is always empty. Is it supposed to be updated by streams statuses?

Reply:
It shouldn't be there; I remove it.
It could be used in case more streams to have detailed info on response, but in Kahpp the stream it's always one